Panels/tests/BreadcrumbDecorator-specs.js

```diff
@@ -130,9 +130,9 @@ describe('BreadcrumbDecorator', () => {
 
 		// tests for truncated {config.max} aria-owns entries in the format ${id}_bc_{$index}
 		const expected = 'test_bc_2';
-		const actual = container.children.item(0).children.item(0).children.item(0);
+		const newActual = screen.getByLabelText('GO TO PREVIOUS');
```

Comment:
```suggestion
		const view = screen.getByLabelText('GO TO PREVIOUS');
```

Reply:
Does this throw warning?

Reply:
No, I must have confused this as a problem instead of container usage.